EasyReflectometryApp/Logic/Proxies/State.py

```diff
@@ -78,12 +81,17 @@ def _onStateChanged(self, changed=True):
 
     @Slot()
     def resetState(self):
-        pass
-        # Need to be reimplemented for EasyReflectometry
-        # self._project_info = self._defaultProjectInfo()
-        # self.projectCreated = False
-        # self.projectInfoChanged.emit()
-        # self._project_proxy.project_save_filepath = ""
-        # self.removeExperiment()
-        # self.resetUndoRedoStack()
-        # self.stateChanged.emit(False)
+        # self.parent._interface = InterfaceFactory()
+        self.parent._project_proxy._project_created = False 
+        self.parent._project_proxy._project_info = self.parent._project_proxy._defaultProjectInfo()
+        self.parent._project_proxy.projectInfoChanged.emit()
```

Comment:
Since there are more than 1-2 project-related reset operations, maybe it would make sense to offload this functionality to project proxy as a separate `reset` method, like you are already doing with data and model proxies?